pkg/pool-weighted/contracts/smart/InvestmentPool.sol

```diff
@@ -72,56 +80,88 @@ contract InvestmentPool is BaseWeightedPool, ReentrancyGuard {
         uint256[] startWeights,
         uint256[] endWeights
     );
-    event SwapEnabledSet(bool swapEnabled);
+    event SwapEnabledChanged(bool swapEnabled);
```

Comment:
```suggestion
    event SwapEnabledSet(bool swapEnabled);
```

I'd like to maintain this as `SwapEnabledSet` for consistency with LBPs if possible. Having two events which are semantically the same could be annoying for subgraph purposes in future when I'm deduplicating some of the data source templates.

Reply:
Ah fair point, I didn't remember the LBPs also had this event - I went with 'changed' as that is what we use in other events. I'll revert this.